ui/shared/components/panel/variants/Pathogenicity.jsx

```diff
@@ -24,17 +35,23 @@ const HGMD_CLASS_NAMES = {
 const hgmdName = hgmdClass => HGMD_CLASS_NAMES[hgmdClass]
 
 
-const PathogenicityLabel = ({ clinsig, formatName }) =>
-  <Label color={CLINSIG_COLOR[CLINSIG_SEVERITY[clinsig]] || 'grey'} size="medium" horizontal basic>
+const PathogenicityLabel = ({ clinsig, formatName, goldStars }) =>
+  <Label color={CLINSIG_COLOR[CLINSIG_SEVERITY[clinsig.toLowerCase()]] || 'grey'} size="medium" horizontal basic>
     {formatName ? formatName(clinsig) : clinsig}
+    {(goldStars != null) && [
+      <HorizontalSpacer key={-1} width={10} />,
+      [0, 1, 2, 3].map(i => (i < goldStars ?
```

Comment:
This looks kind of messy. I know we started with the `Rating` component and then switched to this, but I think in the end it would be neater to use that. Since you are custom styling the color and padding anyways, why not style it on a component that does all this for you